examples/Content.bgl

```diff
@@ -0,0 +1,10 @@
+-- a game to verify that the Content type is created and useable
```

Comment:
This gets parsed and type checked by our tests kills two birds with one stone (ensuring that the type checker and parser both accept Content).

Reply:
A true functional test